covid_graphs/covid_graphs/scripts/server.py

```diff
@@ -56,20 +36,27 @@ def home():
     def covid19_redirect():
         return redirect(url_for("covid19_predictions", event="apr12", graph_type="normal"))
 
-    i = inotify.adapters.InotifyTree(str(data_dir))
-
-    p = Process(target=server.run, kwargs=dict(host="0.0.0.0", port=8081))
+    i = adapters.InotifyTree(str(data_dir),
+                             mask=(constants.IN_MODIFY | constants.IN_DELETE
+                                   | constants.IN_CREATE))
+    p = Process(target=_run_flask_server, kwargs=dict(server=server, data_dir=data_dir))
     p.start()
     while (True):
         events = list(i.event_gen(yield_nones=False, timeout_s=1))
         if len(events) != 0:
             print("Data changed. Restarting server...")
             p.terminate()
             p.join()
-            p = Process(target=server.run, kwargs=dict(host="0.0.0.0", port=8081))
+            p = Process(target=_run_flask_server, kwargs=dict(server=server, data_dir=data_dir))
             p.start()
             print("Server restarted.")
-        sleep(60)
+        sleep(10)
+
+
+def _run_flask_server(server: Flask, data_dir: Path):
+    _create_prediction_apps(server=server, data_dir=data_dir)
+    _create_simulation_apps(server=server, data_dir=data_dir)
```

Comment:
It seems inconsistent that these two have moved but `home()` and `covid19_redirect()` have stayed.

Reply:
TL;DR: Seems like too much price to pay for a bit more consistency.

The callbacks don't have to be updated, AFAIK. To be consistent, we would have to do a move one way or the other. Moving e.g. `home()` is worse perfomance-wise (and readability as well).

The opposite move would require some refactoring. And actually fixing the restart wasn't even my goal with this PR, I just noticed that the fix from https://github.com/lukipuki/COVID-19-simulation/issues/25 wasn't working and was affecting my PR too. Someone else can make this nicer if they want.